recipe/meta.yaml

```diff
@@ -41,67 +37,73 @@ requirements:
     - m2-texinfo               # [win]
     - m2-curl                  # [win]
     - m2-p7zip                 # [win]
-    - readline                 # [not win]
-    - ncurses >=5.9*           # [not win]
+    - readline 6.2*            # [not win]
+    - ncurses 5.9*             # [not win]
     - gcc                      # [not win]
-    - jpeg 8d*                 # [not win]
+    - jpeg 9*                  # [not win]
```

Comment:
I changed all the pinnings to be consistent with conda-forge as much as possible so we can install this alongside the rest of the stack.